WootricSDK/WootricSDK/WTRiPADSocialShareView.m

```diff
@@ -147,7 +154,8 @@ - (void)setXConstraintsForFacebook:(BOOL)facebookAvailable twitter:(BOOL)twitter
       _twitterXConstraint.constant = -(totalWidth / 2) + socialButtonWidth;
```

Comment:
Instead of this:
```
   if (twitterAvailable && facebookAvailable) {
     _facebookLikeXConstraint.constant = 64;
     _twitterXConstraint.constant = 0;
     _facebookXConstraint.constant = -64;
  } else if (!twitterAvailable && facebookAvailable) {
     _facebookLikeXConstraint.constant = 32;
      _facebookXConstraint.constant = -32;
    } else if (!twitterAvailable && !facebookAvailable) {
      _socialShareQuestionLabel.hidden = YES;
```
you could keep the if/else conditions, but move the changes to some methods, something like:
```
   if (twitterAvailable && facebookAvailable) {
     [self twitterAndFacebookAvailableConstraints];
  } else if (!twitterAvailable && facebookAvailable) {
     [self twitterAvailableNotFacebookConstraints];
    } else if (!twitterAvailable && !facebookAvailable) {
      [self twitterAndFacebookUnavailableConstraints];
   }
```

This could be the first step, when you detect more similar functionality you could move to a more explicit method or even to a class.